tools/include/vm_parser.hpp

```diff
@@ -106,9 +106,9 @@ struct VmParamaters {
 
                     if (strncmp(arg, "tr3200", 6) == 0 || strncmp(arg, "TR3200", 6) == 0) {
                         cpu = CpuToUse::TR3200;
-                    }/* else if (strncmp(arg, "DCPU-16N", 6) == 0 || strncmp(arg, "dcpu-16n", 6) == 0) {
+                    } else if (strncmp(arg, "DCPU-16N", 6) == 0 || strncmp(arg, "dcpu-16n", 6) == 0) {
```

Comment:
"DCPU-16N" is a string of length 8, not 6. not that this is a very good way to check arguments mind you.

Reply:
Ha good call out

Reply:
Changed to 8 for the time.